openlibrary/tests/core/test_db.py

```diff
@@ -227,3 +263,37 @@ def test_update_username(self):
         Observations.update_username("@kilgore_trout", "@anonymous")
         assert len(list(self.db.select("observations", where=before_where))) == 0
         assert len(list(self.db.select("observations", where=after_where))) == 1
+
+        assert (
+            len(
+                list(
+                    self.db.select(
+                        "community_edits_queue", where={"submitter": "@kilgore_trout"}
+                    )
+                )
+            )
+            == 2
+        )
```

Comment:
Caution because `assert` is not a function in Python and when parens are used, it is deceptively easy to make [`assert` statements that are always `True`](https://adamj.eu/tech/2020/06/18/why-does-python-syntaxwarning-for-assertion-is-always-true).
```suggestion
        result = self.db.select("community_edits_queue", where={"submitter": "@kilgore_trout"})
        assert len(list(result)) == 2
```

Reply:
Thanks Chris!  Not sure how I missed this, but I've opened #6972 in order to address this.